terminal_table/terminal_table.py

```
@@ -137,7 +137,8 @@ def get_wrapped_row(row, max_column_widths):
 
     @staticmethod
     def get_row_height(wrapped_row):
-        return max(len(col) for col in wrapped_row)
+        max_heights = [len(col) for col in wrapped_row]
```

this crashes on empty list, i wondered if it should return 0 on empty list? I can undo if you don't agree

---

Good find, I'll take a look. I'd expect a get method to return what it says it will return. So `0` would work. On the other I'm not sure if it should accept an empty list at all, it may be better to throw an error.